pkg/checks/traceroute/traceroute.go

```diff
@@ -126,44 +176,69 @@ func TraceRoute(ctx context.Context, cfg tracerouteConfig) (map[int][]Hop, error
 	var wg sync.WaitGroup
 
 	for ttl := 1; ttl <= cfg.MaxHops; ttl++ {
+		c, hopSpan := tr.Start(ctx, fmt.Sprintf("Hop %d", ttl), trace.WithAttributes(
```

Comment:
The most important information about the hop is the address. Lets use the address (ip + port) here instead of `Hop X`.